docs/dev/test-configuration-options.md

```diff
@@ -443,6 +443,26 @@ For virtual device mobile tests, the capability is `deviceOrientation`, but for
 ```
 
 ---
+
+
+---
+### `setupDeviceLock`
+<p><small>| BOOLEAN | <span className="sauceDBlue">Real Devices Only</span> |</small></p>
```

Comment:
@kristofmuhi 
I can imagine situations where people need to test an E2E flow and the passcode needs to be submitted during that flow. Can we also provide which passcode we set?

Reply:
that could make sense, but not in the scope of this MR documentation for the current behavior. 

Didnt hear about this usecase so far from our current customer base who use keyguard for app launches.

Reply:
Why not, it's a few words extra, unless we randomize it